gui/WorldView.cxx

```diff
@@ -1074,14 +1076,25 @@ void WorldView::setLandmarks(kwiver::vital::landmark_map const& lm)
     maxObservations = qMax(maxObservations, observations);
     minZ = qMin(minZ, pos[2]);
     maxZ = qMax(maxZ, pos[2]);
+    zvals.push_back(pos[2]);
+  }
+  if (!zvals.empty())
+  {
+    // Set the range to cover the middle 90% of the data
+    std::sort(zvals.begin(), zvals.end());
+    auto idx = zvals.size() / 20;
```

Comment:
Not sure if I just didn't notice before, but see prior comments re: abbreviations.